WalletWasabi/Microservices/Binaries/README.md

```diff
@@ -7,7 +7,7 @@
 	`git update-index --chmod=+x bitcoind`
 	`git update-index --chmod=+x .\lin64\Tor\tor`
 	`git update-index --chmod=+x .\win64\Tor\tor.exe`
-	`git update-index --chmod=+x .\osx64\Tor\tor.real`
```

Comment:
What is this change about? Changed filename because of Tor Browser release vs Tor release?